NOTICE

```diff
@@ -2,3 +2,11 @@ Morgan Stanley ComposeUI
 Copyright 2021 Morgan Stanley
 
 This product includes software developed at Morgan Stanley (http://www.morganstanley.com).
+
+=======================================================================
+3rd party software used by ComposeUI
```

Comment:
Did Legal request/review this?

Reply:
What are we going to do about it? Should I do anything?